src/openforms/registrations/utils.py

```diff
@@ -7,13 +7,15 @@
 unset = object()
 
 
-def execute_unless_result_exists(
-    callback: Callable,
+def execute_unless_result_exists[
```

Comment:
Could you please elaborate a bit?I haven't seen this structure

Reply:
it's new syntax available in python 3.12!

You can specify the generic typevar between square brackets now, instead of having to do:

```py
R = TypeVar("R")

def execute_unless_result_exists(callback: Callable[[], R]) -> R:
    ...